.github/workflows/tests.yaml

```diff
@@ -0,0 +1,44 @@
+on:
+    push:
+      branches:
+        - main
+    pull_request:
+      branches:
+        - main
+  
+jobs:
+    setup-and-run:
+      runs-on: ubuntu-latest
+  
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v3
+  
+        - name: Set up Docker
+          uses: docker/setup-buildx-action@v2
+  
+        - name: Install Aztec CLI
+          run: |
+            curl -s https://install.aztec.network > tmp.sh
+            bash tmp.sh <<< yes "yes"
+  
+        - name: Update path
+          run: echo "/home/runner/.aztec/bin" >> $GITHUB_PATH
+  
+        - name: Set Aztec version and start sandbox
+          run: |
+            VERSION=0.68.0 aztec-up
```

Comment:
Instead of hardcoding 0.68.0, maybe we can fetch the latest release using something like this step? https://github.com/AztecProtocol/aztec-starter/blob/b804e994a65d8a353db5345aa7ec030f0b9b1537/.github/workflows/update.yaml#L22

Reply:
hmm, i was thinking to add it to the `update_contract.sh` script, but then figured out that is a tool that you use to push the new version

should we fetch the latest version, it'd be incompatible with the `package.json` one, since the latest one right now is `0.68.2`, what do you think? 

options are:
- expect the dev to manually update the workflow (doing global search for `0.68.0` and replace for `0.68.2`)
- add a snippet in the `update_contract.sh` that updates the workflow file
- add an api call in the workflows file, that may differ with the `package.json` versions

will try to implement the third one

Reply:
**update**: implemented it here https://github.com/defi-wonderland/aztec-starter/pull/2, had to upgrade the `package.json` versions as well, because having a non-matching version if aztec-up vs the imported packages makes the accounts "not being registered" error.

wdyt @critesjosh? unless we dynamically create the package json, we'll be always facing this issue where a test could start failing since a new version was released, i'm not pleased about that

Reply:
update: reverted the package.json changes by pulling the updated `main` up branch